webrender/src/texture_cache.rs

```diff
@@ -581,7 +581,14 @@ impl TextureCache {
                     height: TEXTURE_LAYER_DIMENSIONS,
                     format: descriptor.format,
                     filter: texture_array.filter,
-                    render_target: None,
+                    // TODO(gw): Creating a render target here is only used
+                    //           for the texture cache debugger display. In
+                    //           the future, we should change the debug
+                    //           display to use a shader that blits the
+                    //           texture, and then we can remove this
+                    //           memory allocation (same for the other
+                    //           standalone texture below).
+                    render_target: Some(RenderTargetInfo { has_depth: false }),
```

Comment:
I like that this is fixed, but I don't quite like the fact that are paying for the texture debug scenario in all the cases where debug is not needed.

Reply:
I agree - it's a quick fix for now, but a proper fix would be helpful. I'll add help-wanted tag to the issue @nical opened for it.